include/rmm/logger.hpp

```diff
@@ -43,9 +43,11 @@ inline std::string default_log_filename()
   return (filename == nullptr) ? std::string{"rmm_log.txt"} : std::string{filename};
 }
 
-// Simple wrapper around a spdlog::logger that performs RMM-specific initialization
+/**
+ * @brief Simple wrapper around a spdlog::logger that performs RMM-specific initialization
+ */
 struct logger_wrapper {
-  spdlog::logger logger_;
+  spdlog::logger logger_;  ///< The underlying logger
```

Comment:
Duh.  Perhaps if this were a private member Doxygen wouldn't complain about it not having an obvious docstring? Is Doxygen against POD? :)